docs/lib/datastore/fragments/js/sync/50-selectiveSync.md

```diff
@@ -82,6 +92,11 @@ DataStore.configure({
   ]
 });
 ```
+<amplify-callout warning>
+
+`DataStore.configure` should only by called once at the root of your file.
```

Comment:
add parenthesis to configure. 

Root of your file? Not root of your app's lifecycle?

Reply:
> Not root of your app's lifecycle?

Technically, yes. Just wasn't sure how to phrase that in a way that's not confusing